### PR TITLE
Generalise and revise filestore page (ensure applies to ShARC too)

### PR DESCRIPTION
**Pull request by willfurnass**
Wanted to add links from the jupyterhub docs to docs on ShARC's storage but realised we didn't have any; it was all Iceberg-specific.  This PR is an attempt to
 - better enumerate the storage options available to users
 - add more info on pros and cons
 - explicitly Iceberg-specific only where needs to be
 - rewrite to use git-friendly semantic line endings.

### CODE REVIEW

File: hpc/filestore.rst [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+* as a subdirectory of ``/shared`` on ShARC/Iceberg (you need to **explicitly request HPC access when you order storage from CiCS**).
+  
+Note that this subdirectory will be **mounted on demand** on ShARC/Iceberg: 
+it will not be visible if you simply list the contents of the ``/shared`` directory but 
+will be accessible if you ``cd`` (change directory) into it e.g. ``cd /shared/my_group_file_share1``
+
+**Regarding permissions**: 
+behind the scenes, the file server that provides this shared storage manages permissions using 
+Windows-style `ACLs <https://en.wikipedia.org/wiki/Access_control_list>`_ 
+(which can be set by area owners via *Group Management* web interface).
```
Comment (anthonybrookfield) [inline hpc/filestore.rst]: link directly to webpages? 
 `Research storage management web interface <sheffield.ac.uk/storage>`__

File: hpc/filestore.rst [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+  unless Perl is explicitly told to test for file permissions in a more thorough way 
+  (see the mention of ``use filetest 'access'`` `here <http://perldoc.perl.org/functions/-X.html>`_).
+* git: may complain that permissions have changed if 
+  a repository is simply moved to ``/shared/someplace`` from elsewhere on ShARC/Iceberg.  
+  As a workaround you can tell git to not to track Linux permissions for a single repository using 
+  ``git config core.filemode false`` or 
+  for all repositories using ``git config --global core.filemode false``.
+
+
+``/scratch``: for reading/writing small files
```
Comment (anthonybrookfield) [inline hpc/filestore.rst]: use the ``/path`` style for each of the other filestore sections too for consistancy?

File: hpc/filestore.rst [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+    **older files** on /fastdata are **automatically deleted**: 
+    a report of files older than 60 days is regularly generated, 
+    the owners of these files are then notified by email then 
+    a week after the email(s) are sent the identified files are deleted. 
+
+    We reserve the right to change this policy without warning in order to ensure efficient running of the service.
+
+    It is important to therefore not use /fastdata for long-term storage and 
+    **copy important data on ``/fastdata`` to backed-up areas such as ``/home``, ``/data`` or ``/shared``**.
+
```
Comment (anthonybrookfield) [inline hpc/filestore.rst]: I think the ``/paths`` need to escape from the ** ** to render properly

File: hpc/filestore.rst [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+    chmod 755 /fastdata/yourusername
+    chmod 755 /fastdata/yourusername/public
+    chmod 700 /fastdata/yourusername/private
+
+The fastdata area provides **669 TeraBytes** of storage on SHARC and **260 Terabytes** of storage on Iceberg.
+It takes advantage of the internal high-speed network infrastructure (OmniPath interconnects for ShARC; Infiniband for Iceberg) for fast access to data.
+
+``/fastdata`` is **optimised for large file operations**.
+it is faster than ``/home/``, ``/data`` and '``/shared``' (see below) when dealing with larger files but
```
Comment (anthonybrookfield) [inline hpc/filestore.rst]: stray quotes

File: hpc/filestore.rst [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+
+You can use the ``lfs``  command to find out which files under ``/fastdata`` are older than a certain number of days and hence approaching the time of deletion. 
+For example, to find files 50 or more days old ::
+
+    lfs find -ctime +50 /fastdata/yourusername
+
+
+**Backup policy:** ``/fastdata`` is **not backed up**.
+The next best I/O performance that requires the minimum amount of work is achieved by 
+keeping your data in the ``/fastdata`` area and
```
Comment (anthonybrookfield) [inline hpc/filestore.rst]: .....? This line seems to terminate abruptly

**Comment by willfurnass** [comment]
Hmm, Travis can't build the PDF:
```
...
LaTeX Warning: Hyper reference `sharc/index:sharc' on page 1 undefined on input

 line 59.

LaTeX Warning: Hyper reference `iceberg/index:iceberg' on page 1 undefined on i

nput line 59.

[1 <./iceberg.png (PNG copy)>] [2]

Chapter 1.

[3] [4]

Chapter 2.

(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ts1ptm.fd) [5]

<mobaxterm-welcome.png, id=71, 1157.32375pt x 867.24pt>

<use mobaxterm-welcome.png>

LaTeX Warning: Hyper reference `hpc/connecting:ssh' on page 6 undefined on inpu

t line 147.

<mobaxterm-terminal.png, id=72, 1156.32pt x 867.24pt>

<use mobaxterm-terminal.png> [6 <./mobaxterm-welcome.png> <./mobaxterm-terminal

.png>]

LaTeX Warning: Hyper reference `hpc/connecting:ssh' on page 7 undefined on inpu

t line 164.

(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1pcr.fd)

LaTeX Warning: Hyper reference `hpc/scheduler/sge:sge-queue' on page 7 undefine

d on input line 237.

LaTeX Warning: Hyper reference `sharc/software/index:sharc-software' on page 7 

undefined on input line 239.

LaTeX Warning: Hyper reference `iceberg/software/index:iceberg-software' on pag

e 7 undefined on input line 239.

[7] [8]

! Package inputenc Error: Unicode char \u8:  not set up for use with LaTeX.
...
```

**Pull request merged**